test/print-config-helpers.js

```diff
@@ -0,0 +1,128 @@
+const { spawnSync } = require('child_process')
+const os = require('os')
+
+const nodePath = process.execPath
+const isWin = (os.platform() === 'win32')
+const pwd = process.cwd()
+const whiteSpaceReg = /[\\\s]/g
+const pwdReg = new RegExp(pwd, 'g')
+
+/**
+ * Function: runSpawn
+ *
+ * @param {Array} args: array of arguments to pass
+ *  to child spawned process.
+ * @param {Boolean} text=false: expect json or cli
+ *  text to be returned.
+ * @param {Boolean} stripWhiteSpace=false: should all
+ *  whitespace be removed from the `out` string?
+ * @returns {String} out: a string representing the stdout
+ *   of the child process
+ *
+ * A wrapper function around spawnSync.
+ * Node.js Docs: http://tinyurl.com/66ztyx9b
+ *
+ * Todo:  Whitespace for snapshots is showing up in
+ * different amounts on different os platforms.
+ *
+ * I am concern about this issue in the chai-snapshot
+ * package.
+ *
+ * Issue described in jest
+ * https://github.com/jestjs/jest/pull/9203
+ *
+ * Last time chai-jest-snapshot was publish was 6
+ * years ago.
+ *
+ * https://www.npmjs.com/package/chai-jest-snapshot?activeTab=versions
+ *
+ * Alternative packages that might work.
+ *
+ * https://www.npmjs.com/package/mocha-chai-jest-snapshot
+ *
+ * https://www.npmjs.com/package/chai-snapshot-matcher/v/2.0.2
+ *
+ *
+ */
+const runSpawn = (args, text = false, stripWhiteSpace = false) => {
+  const doubleQuoteReg = /"/g
+  const slashReg = /\\/g
+
+  const { output } = spawnSync(nodePath, args)
+
+  let out = output.toString('utf8')
+
+  if (isWin && text) {
+    const jsonEncodedPwd = JSON.stringify(pwd)
+      .replace(doubleQuoteReg, '')
+    const encodedPwdRegEx = new RegExp(jsonEncodedPwd, 'g')
+    out = out.replace(encodedPwdRegEx, '.')
+  } else if (isWin && !text) {
+    const jsonEncodedPwd = JSON.stringify(pwd)
+      .replace(doubleQuoteReg, '')
+      .replace(slashReg, '\\\\')
+    const encodedPwdRegEx = new RegExp(jsonEncodedPwd, 'g')
+    out = out.replace(encodedPwdRegEx, '.')
+  } else if (!isWin) {
+    out = out.replace(pwdReg, '.')
+  }
+
+  // Todo: For certain cases we need to strip out all whitespace in
+  // snapshots. See notes above
+  if (text && stripWhiteSpace) { out = out.replace(whiteSpaceReg, '') }
+
+  if (!text) { out = cleanJson(out) }
```

Comment:
use `// eslint-disable-line` for single line if statements.  Try to find a specific eslint rule.  I think the rule is on [this eslint](https://eslint.org/docs/latest/rules/curly) docs page.  To disable a single line, refer to [this page](https://www.npmjs.com/package/standard#how-do-i-disable-a-rule) on standard docs.